packages/fastauth-metamask/src/lib/lib.tsx

```diff
@@ -0,0 +1,399 @@
+import React from 'react'
+import { ethers } from "ethers";
+import { createRoot } from "react-dom/client";
+import detectEthereumProvider from "@metamask/detect-provider";
+import * as nearAPI from "near-api-js";
+import { Modal } from "./modal";
+import { generateSeedPhrase } from "near-seed-phrase";
+
+// constants
+
+const domain = {
+	name: "NETH",
+	version: "1",
+	// chainId: 1, // aurora
+	chainId: 1313161554, // aurora
+};
+const MODAL_ELEMENT_ID = "near-wallet-selector-modal";
```

Comment:
This ID conflicts with the main modal https://github.com/near/wallet-selector/blob/main/packages/modal-ui/src/lib/modal.tsx#L12 how is this meant to be used?